peps/pep-0770.rst

```diff
@@ -179,98 +204,62 @@ Specification
 
 The changes necessary to implement this PEP include:
 
-* Additions to `Core Metadata <770-spec-core-metadata_>`_, as defined in the
-  `Core Metadata specification <coremetadataspec_>`__.
-* Additions to the author-provided
-  `project source metadata <770-spec-project-source-metadata_>`_, as defined in the
-  `pyproject.toml specification <pyprojecttoml_>`__.
-* `Additions <770-spec-project-formats_>`_ to the source distribution (sdist),
-  built distribution (wheel), and installed project specifications.
+* A new reserved registry of subdirectory names in the ``.dist-info`` directory.
```

Comment:
If I'm understanding correctly, the idea is that all names in `.dist-info` are reserved, and this registry is the ones that are allowed? If so, this wasn't entirely clear from the text, as I thought that some names might be unreserved. Perhaps?:
```suggestion
* Explicitly reserving all names  in the ``.dist-info`` directory.
* A new registry of reserved subdirectory names in the ``.dist-info`` directory.
```